migrations/5_test_case_id_type.up.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE test_item
+    ALTER COLUMN test_case_id TYPE VARCHAR(256);
+
+ALTER TABLE test_item
+    ADD COLUMN test_case_hash INTEGER;
+
+CREATE INDEX test_case_hash_idx ON test_item (test_case_hash);
+CREATE INDEX test_case_hash_launch_id_idx ON test_item (test_case_hash, launch_id);
```

Comment:
do we really need this index

Reply:
check what indexes are used in history 
can be used queries for getting indexes usage https://wiki.postgresql.org/wiki/Index_Maintenance

Reply:
Yep, this index is used pretty often, tested on my local instance
![image](https://user-images.githubusercontent.com/31598109/70428508-c23e6e80-1a87-11ea-94a9-cbd7a37af612.png)